spec/filters/kv_spec.rb

```diff
@@ -730,6 +730,39 @@
     end
   end
 
+  describe "trim_key/trim_value options : trim multiple matching characters from either end" do
+    subject do
+      plugin = LogStash::Filters::KV.new(options)
+      plugin.register
+      plugin
+    end
+
+    let(:message) { "to=<foo@example.com>, orig_to=<bar@example.com>, %+relay=mail.example.com[private/dovecot-lmtp], delay=2.2, delays=1.9/0.01/0.01/0.21, dsn=2.0.0, status=sent (250 2.0.0 <foo@example.com> YERDHejiRSXFDSdfUXTV Saved) " }
+    let(:data) { {"message" => message} }
+    let(:event) { LogStash::Event.new(data) }
+    let(:options) {
+      {
+          "field_split" => " ",
+          "value_split" => "=",
+          "trim_value" => "<>,",
+          "trim_key" => "%+"
+      }
+    }
+
+    context '' do
```

Comment:
since there's no context here, can we just move the `it` block one level back?

Reply:
d'oh, will fix this and do the merge on Monday.